react-app/src/containers/BaseScreen/BaseScreen.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+import './BaseScreen.css';
+
+export default class BaseScreen extends Component {
+  render() {
+    return (
+      <div className="BaseScreen">
+        <div className="header">
+          {this.props.header}
```

Comment:
Isn't it confusing to have a `header` className and a `Header` one?

Reply:
yes, situations like this could cause collision due to the global namespacing nature of css.
We will address this using css modules, introduced here: https://github.com/Unixono/beautifulthings/pull/16

Reply:
Done.